react-delegationdashboard/src/components/Layout/APRCalculation.ts

```diff
@@ -27,20 +35,30 @@ const calculateAPR = ({
   blsKeys: ContractReturnData[];
   totalActiveStake: string;
 }) => {
-  const feesInEpoch = 0;
-  const stakePerNode = 2500;
-  const protocolSustainabilityRewards = 0.1;
+  const allNodes = blsKeys.filter(key => key.asString === 'staked' || key.asString === 'jailed')
+    .length;
+  const allActiveNodes = blsKeys.filter(key => key.asString === 'staked').length;
+  if (allActiveNodes <= 0) {
+    return '0.00';
+  }
+
+  const daysInYear = 365;
```

Comment:
in mainnet we have one epoch = one day
but in testnet/devnet we might have different configs, (currently testnet has 4 hour epochs)
We might want to change from daysInYear to epochsInYear, and have this computed based on 
erd_round_duration (milliseconds) and erd_rounds_per_epoch, then the APR will be correctly computed also for 

erd_round_duration you can get from network/config but erd_rounds_per_epoch is currently only available in network/status/<shardid> (but will be moved as well in network/config)

Reply:
done